content/docs/started/getting-started.md

```diff
@@ -49,15 +49,15 @@ export KUBEFLOW_VERSION=0.2.2
 curl https://raw.githubusercontent.com/kubeflow/kubeflow/v${KUBEFLOW_VERSION}/scripts/deploy.sh | bash
 ```
 
-For more detailed instructions about how to use Kubeflow please refer to the [user guide](/docs/about/user_guide/).
+For more detailed instructions about how to use Kubeflow please refer to the [guide](/docs/about/guides/).
```

Comment:
This link doesn't appear to work in the preview.
I think it should be
/docs/guides/

Reply:
@0snug0 Thanks for helping out with this.
This broken link should be resolved now based on PR #119.